third_party/wb_category_select/ft.wb_category_select.php

```diff
@@ -370,12 +370,17 @@ private function _get_category_data($cat_ids)
 		$parse = array();
 		foreach ($cat_ids as $category_id)
 		{
-			$parse[] = array(
-				'category_id'        => $category_id,
-				'category_name'      => $category_data[$category_id]['cat_name'],
-				'category_url_title' => $category_data[$category_id]['cat_url_title'],
-				'category_image'     => $category_data[$category_id]['cat_image']
-			);
+			if( isset( $category_data[$category_id] ) )
+			{
+				$process = array();
+				foreach ($category_data[$category_id] as $k => $v)
+				{
+					$k = false !== strpos($k,'cat_') ? str_replace('cat_','category_',$k) : 'category_'.$k;
```

Comment:
Change the order of the conditional (I'm not a fan of Yoda conditionals) and wrap the conditional in a parentheses.